doc/superbuild/source/developers/releases/Checklist.rst

```diff
@@ -118,22 +137,13 @@ web pages.
       * Edit Main Page:
         https://computing-staging.llnl.gov/projects/sundials
 
-        * Update Release History page:
-          https://computing-staging.llnl.gov/projects/sundials/release-history
-        * Add list of primary changes and child page containing
-          full list of changes (this information is shared with
-          "Changes in ..." sections of user guides).
-
       * Edit Download Page:
         https://computing-staging.llnl.gov/projects/sundials/sundials-software
 
         * Update main download table with links to new versions of solvers
         * The documentation links do not need to be updated (they point to the PDFs on the GitHub main branch)
```

Comment:
I'm guessing the link for the example documentation PDFs needs to be updated now that we upload them with the tarballs.